plugins/pastefromword/filter/default.js

```diff
@@ -542,6 +528,24 @@
 			element.attributes.style = CKEDITOR.tools.writeCssText( styles );
 		},
 
+		removeStyles: function( element, styles ) {
+			CKEDITOR.tools.array.forEach( styles, function( style ) {
+				Style.setStyle( element, style, null );
+			} );
+		},
+
+		convertStyleToPx: function( element ) {
```

Comment:
Not sure if I understand, as I know `CKEDITOR.tools.convertToPx` does more or less what you said, so how this function is different?

https://github.com/ckeditor/ckeditor-dev/blob/015e1cc00b114d6a503421db9c0a35b71afe525e/plugins/pastefromword/filter/default.js#L537-L547

Reply:
<img width="257" alt="Difference in sizes between 50pt (bigger) and 50px (smaller)" src="https://user-images.githubusercontent.com/1078728/52337573-4a875980-2a08-11e9-8907-7a5970a3489b.png">

Converting in this way we change the actual size of the text. To get the real size we need to get the size of pt in pixels. To do this we must use `getComputedStyle` on element, which has styles declared in pt. However it can be extracted to the separate issue.

Reply:
I'm not sure if I understand the issue. As you can see this function uses `CKEDITOR.tools.convertToPx` under the hood, so it won't give `50pt` -> `50px` but `50pt` -> `67px`. You can test it here: https://codepen.io/jacekbogdanski/pen/rProxW

What's wrong with that?